terraform/helm/aptos-node/values.yaml

```diff
@@ -35,8 +35,8 @@ validator:
     provide_genesis: true
     sync_only: false
     concurrency_level: 8
-    round_initial_timeout_ms:
-    quorum_store_poll_count: 1
+    round_initial_timeout_ms: 1000
```

Comment:
Can you clarify why is this needed? Its already set to 1000 as default right?

Reply:
what does empty mean? should it be removed? I can leave it as is.

Reply:
Yes, I think we can just remote it.